src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,61 @@
+import styles from "./Pagination.module.css";
+
+function Pagination({
+  currentPage,
+  totalPages,
+  onPageChange,
+  maxPageButtons = 5,
+}) {
```

Comment:
매직 넘버를 사용하지않고, 상수로 분리해보면 어떨까요?
```suggestion
const GROUP_SIZE = 5;

function Pagination({
  currentPage,
  totalPages,
  onPageChange,
  maxPageButtons = GROUP_SIZE,
}) {
```

이렇게 매 렌더링때마다 재계산 될 필요없는 상수의 경우, 컴포넌트 외부로 빼두면 상수가 한 번만 생성되고 재사용되므로 메모리 사용을 더 효율적으로 할 수 있답니다 :)

Reply:
Pagination 컴포넌트 외부에서 재사용될 일이 거의 없을 거라고 생각되는 상수라서  constants.js로 빼지는 않고 가르쳐주신대로 컴포넌트 파일 안에 정의하였습니다.